server/cli/cli.go

```diff
@@ -63,57 +63,59 @@ Output build version information.
 	},
 }
 
-var allCmds = &commander.Commander{
-	Name: "cockroach",
-	Commands: []*commander.Command{
-		// Node commands.
-		initCmd,
-		startCmd,
-		exterminateCmd,
-		quitCmd,
-
-		// Certificate commands.
-		createCACertCmd,
-		createNodeCertCmd,
-
-		// Key/value commands.
-		getCmd,
-		putCmd,
-		incCmd,
-		delCmd,
-		scanCmd,
-
-		// Range commands.
-		lsRangesCmd,
-		splitRangeCmd,
-		mergeRangeCmd,
-
-		// Accounting commands.
-		getAcctCmd,
-		lsAcctsCmd,
-		rmAcctCmd,
-		setAcctCmd,
-
-		// Permission commands.
-		getPermsCmd,
-		lsPermsCmd,
-		rmPermsCmd,
-		setPermsCmd,
-
-		// Zone commands.
-		getZoneCmd,
-		lsZonesCmd,
-		rmZoneCmd,
-		setZoneCmd,
-
-		// Miscellaneous commands.
-		// TODO(pmattis): stats
-		listParamsCmd,
-		versionCmd,
-	},
+var allCmds = []*cobra.Command{
```

Comment:
yep, see https://github.com/cockroachdb/cockroach/pull/922/files#diff-fd6f83b6931534806d1a150828e05695R118

maybe that should happen on `init()`